EventFilter/SiPixelRawToDigi/plugins/SiPixelRawToDigi.h

```diff
@@ -60,5 +60,6 @@ class SiPixelRawToDigi : public edm::stream::EDProducer<> {
   int nwords;
   bool usePilotBlade;
   bool usePhase1;
+  std::string CablingMapLabel;
```

Comment:
@tvami 
to be closer to cms convention, please make this cablingMapLabel (better would be cablingMapLabel_, but it seems the full class does not distiguish between member data and local variables so it does not make much difference to have the _.

Thanks!